File: Swu.Portal.Web/Scripts/model/video.ts
module Swu {
    export interface IVideo{
        id: number;
        imageUrl: string;
        videoUrl: string;
        title?: string;
        title_en?: string;
        title_th?: string;
        isActive?: boolean;
    }
    export interface IVideoAlbum {
        id: number;
        photos: IVideo[];
    }
}

File: Swu.Portal.Data/Models/Video.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Swu.Portal.Data.Models
{
    public class Video :IEntity
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string ImageUrl { get; set; }
        public string VideoUrl { get; set; }
        public string Title_EN { get; set; }
        public string Title_TH { get; set; }
        [DefaultValue(false)]
        public bool IsActive { get; set; }
    }
}
